packages/terra-framework-docs/src/terra-dev-site/test/data-grid/flowsheet-data-grid/ColumnHeadersHidden.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useCallback } from 'react';
+import { FlowsheetDataGrid } from 'terra-data-grid';
+
+const DefaultFlowsheetDataGrid = () => {
```

Comment:
I believe this should be ColumnHeadersHidden instead of DefaultFlowsheetDataGrid.